templates/engage/shared/_kr_engage_form.html

```diff
@@ -0,0 +1,69 @@
+<!-- MARKETO FORM -->
+<form action="/marketo/submit" method="post" id="mktoForm_{{ id }}">
+  <fieldset class="u-no-padding--top" style="border-bottom-width: 1px;">
+    <legend class="u-off-screen">연락처 정보</legend>
+    <ul class="p-list">
+      <li>
+        <label for="firstName">이름:</label>
+        <input required id="firstName" name="firstName" maxlength="255" type="text" value="">
+      </li>
+      <li>
+        <label for="lastName">성:</label>
+        <input required id="lastName" name="lastName" maxlength="255" type="text">
+      </li>
+      <li>
+        <label for="email">직장 이메일:</label>
+        <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" value="">
+      </li>
+      <li>
+        <label for="company">회사 이름:</label>
+        <input required id="company" name="company" maxlength="255" type="text">
+      </li>
+      <li>
+        <label for="jobTitle">직위:</label>
+        <input required id="jobTitle" name="title" maxlength="255" type="text">
+      </li>
+      <li>
+        <label for="phone">휴대폰 번호:</label>
+        <input required id="phone" name="phone" maxlength="255" type="tel">
+      </li>
+      <li>
+        <label class="p-checkbox">
+          <input class="p-checkbox__input" name="canonicalUpdatesOptIn" aria-labelledby="canonicalUpdatesOptIn" value="yes" type="checkbox">
+          <span class="p-checkbox__label" id="canonicalUpdatesOptIn">Canonical의 제품 및 서비스에 대한 정보 수신에 동의합니다.</span>
+        </label>
+      </li>
+      <li>
+        <p>이 양식을 제출함으로써 나는 다음 내용을 읽었으며 이에 동의함을 확인합니다 <a href="/legal/data-privacy/contact">Canonical의 개인정보 보호정책</a> 그리고 <a href="/legal/data-privacy">개인 정보 정책</a>.
```

Comment:
This way, it will sound more natural in Korean.

<p>이 양식을 제출함으로써 나는 다음과 같은 내용에 동의함을 확인합니다: <a href="/legal/data-privacy/contact">Canonical의 개인정보 보호정책</a> 및 <a href="/legal/data-privacy">개인 정보 정책</a>.